### ISSUE TITLE
config Settings Not Being Honored, bootstrap 4

### ISSUE THREAD

**Issue by temp1029** [ankurk91/vue-bootstrap-datetimepicker#34]
**_This issue could possibly be because I am using BS4, but wanted to see if that's really the case._**

**I'm submitting a ...**  (check one with "x")
```
[X] Bug report => search github for a similar issue or PR before submitting
[ ] Feature request
[ ] Other, please describe
```

**Tell about your platform**
* eonasdan-bootstrap-datetimepicker version : 4.17.47
* Vue.js version : 2.5.16
* Browser name and version : Chrome 66.0.3359.181 64-bit
* This package version :  4.1.3

**Current behavior**
Clear button not showing despite config option.

**Expected behavior**
Clear button should show

**Minimal reproduction of the problem with instructions**
Initilize date-picker, then click in field to open it

**Code:**
![image](https://user-images.githubusercontent.com/4396942/40523802-5c8e416c-5fa5-11e8-9b9f-d764c11fce1d.png)

**Display**
![image](https://user-images.githubusercontent.com/4396942/40523718-011f0410-5fa5-11e8-9f0d-86b1d8095788.png)

**Comment by ankurk91** [comment]
Sorry screenshots does not help, can you set a jsfiddle for the same?

https://jsfiddle.net/ankurk91/zupazg2u/

**Comment by temp1029** [comment]
Thanks for the quick response!

Unfortunately there are some custom styles and (I think) functionality of the theme I'm using. When I just replace BS3 with BS4 in jsFiddle it breaks the entire picker, with nothing being displayed. I suspect these custom settings are whats causing the issue.

Any idea when you'll have Issue #33 done? Not trying to be rude, just want to see if I need to look for another solution or not.

**Comment by ankurk91** [comment]
@temp1029 
Will try to release beta this weekend

**Comment by temp1029** [comment]
Cool, I’ll be happy to beta test it for you if you want.

**Comment by ankurk91** [comment]
Just released v5.0.0-beta.1
```
npm install vue-bootstrap-datetimepicker@next
```
